pkg/ext-proc/backend/pod_metrics_refresher.go

```diff
@@ -0,0 +1,79 @@
+package backend
+
+import (
+	"context"
+	"time"
+
+	klog "k8s.io/klog/v2"
+)
+
+const (
+	// TODO: make it configurable
+	fetchMetricsTimeout = 1 * time.Second
```

Comment:
can you make a flag in main.go for this, similar to the fetchMetricInterval? I think this is important to tune in different environments.

Reply:
Done